src/ZEServices.ts

```diff
@@ -562,6 +562,29 @@ export class ZEServices {
         return this.postJSON<any>(data, this.createPath(accountId, vin) + "/actions/charging-start", country);
     }
 
+    /** Pause/Resume Charging for Dacia Spring.
+     * @param pause: If true, then pause; if false, then resume.
+     * @param accountId The accountId.
+     * @param vin The vehicle identifier.
+     * @param country optional country
+     */
+    async setChargePause(pause: boolean, accountId: string, vin: string, country?: string) {
+        
+        let data: DataContainer<any> =
+        {
+            "data":
+            {
+                "type": "ChargePauseResume",
+                "attributes":
+                {
+                    "action": action ? "pause" : "resume"
```

Comment:
```suggestion
        const data: DataContainer<any> =
        {
            "data":
            {
                "type": "ChargePauseResume",
                "attributes":
                {
                    "action": pause ? "pause" : "resume"
```

Reply:
@JLorenz86 please adapt the current code. It does not compile and I also made a small codesmell fix